components/integrations/bots/add_bot/add_bot.jsx

```diff
@@ -0,0 +1,701 @@
+                                </div>
+                            </div>
+                        </div>
+                        <div className='form-group'>
+                            <label
+                                className='control-label col-sm-4'
+                                htmlFor='postChannels'
+                            >
+                                <FormattedMessage
+                                    id='bot.add.post_channels'
+                                    defaultMessage='post:channels'
+                                />
+                            </label>
+                            <div className='col-md-5 col-sm-8 checkbox'>
+                                <input
+                                    id='postChannels'
+                                    type='checkbox'
+                                    checked={this.state.postChannels || this.state.role === roleOptionSystemAdmin || this.state.postAll}
+                                    onChange={this.updatePostChannels}
+                                    disabled={!this.props.editingUserHasManageSystem || this.state.role === roleOptionSystemAdmin || this.state.postAll}
+                                />
+                                <div className='form__help'>
+                                    <FormattedMessage
+                                        id='bot.add.post_channels.help'
+                                        defaultMessage='Bot will have access to post to all Mattermost public channels.'
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                        <div className='backstage-form__footer'>
+                            <FormError
+                                type='backstage'
+                                errors={[this.state.error]}
+                            />
+                            <Link
+                                className='btn btn-link btn-sm'
+                                to={`/${this.props.team.name}/integrations/bots`}
+                            >
+                                <FormattedMessage
+                                    id='bots.manage.add.cancel'
+                                    defaultMessage='Cancel'
+                                />
+                            </Link>
+                            <SpinnerButton
+                                className='btn btn-primary'
+                                type='submit'
+                                spinning={this.state.adding}
+                                spinningText={buttonActiveText}
+                                onClick={this.handleSubmit}
+                            >
+                                {buttonText}
+                            </SpinnerButton>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        );
+    }
+}
```

Comment:
Can't see translation strings in i18n

Reply:
Yup. Been putting this off to avoid conflicts. Just pushed the strings!